src/simsopt/field/coil.py

```diff
@@ -118,6 +122,45 @@ def vjp(self, v_current):
     def get_value(self):
         return self.scale * self.current_to_scale.get_value()
 
+    def set_dofs(self, dofs):
+        self.current_to_scale.set_dofs(dofs / self.scale)
+
+
+def current_pure(dofs):
+    return dofs
+
+
+class JaxCurrent(sopp.Current, CurrentBase):
```

Comment:
Why does this class exist? Docstring needed

Reply:
Added a docstring. Probably more useful if and when we ever get a Jax BiotSavart object working. Like JaxCurve it is essentially a duplicate.

Reply:
confused by `JaxCurrent`, not sure about its use case.  even with a Jax-based BiotSavart class, I don't think we can Jax through this class ? maybe I'm mistaken